build.gradle.kts

```diff
@@ -76,6 +77,10 @@ tasks {
     }
 
     withType<ShadowJar> {
+        configureEach {
```

Comment:
Måtte legge til dette for at oppstartsklassen skulle finnes (fant noe dokumentason for SpringBoot 2.7). Det er kryptisk, men ser ut til å løse problemet.